.github/workflows/pre-commit.yml

```diff
@@ -57,4 +46,5 @@ jobs:
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
         continue-on-error: true
 
-      - run: nix develop --command stylix-check --no-nom
+      - name: Check Pre-Commit Hooks
+        run: nix build .#checks.${{ matrix.name }}.pre-commit
```

Comment:
This is technically fine, as it's a fixed known value. But directly interpolating github-actions `${{ }}` templates into bash scripts is _generally_ a bad habit to get into.

I'd recommend using the

```yaml
env:
  system: ${{ matrix.name }}
run: nix build ".#checks.$system.pre-commit"
```

pattern, where variables are injected as _shell_ variables using the `env` context.